packages/navigator/src/browser/navigator-filter.ts

```diff
@@ -14,7 +14,7 @@
  * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
  ********************************************************************************/
 
-import { inject, injectable } from 'inversify';
+import { inject, injectable, postConstruct } from 'inversify';
 import { Minimatch } from 'minimatch';
```

Comment:
Module 'minimatch' is not listed as dependency in package.json

Reply:
i will address this comment in a separate PR :)